cranelift-entity/src/boxed_slice.rs

```diff
@@ -27,6 +27,10 @@ where
 {
     /// Create a new slice from a raw pointer. A safer way to create slices is
     /// to use `PrimaryMap::into_boxed_slice()`.
+    ///
+    /// # Safety
+    ///
+    /// relies on `raw` pointing to a valid slice of `V`s
```

Comment:
Before @bnjbvr gets to this: he'll say this should be capitalized and punctuated.